src/styles.css

```diff
@@ -2535,6 +2535,88 @@ body .flairselector li:hover {
 	height: 22px;
 	width: 21px;
 }
+.flair-ahq-esports-club, .md a[href*="ahq-esports-club"] {
+	background-position: -169px -46px;
+	height: 24px;
+	width: 26px;
+}
+.flair-detonator-gold, .md a[href*="detonator-gold"] {
+	background-position: -190px 0;
+	height: 21px;
+	width: 25px;
+}
+.flair-flash-wolves, .md a[href*="flash-wolves"] {
+	background-position: -186px -22px;
+	height: 21px;
+	width: 35px;
+}
+.flair-fireball, .md a[href*="fireball"] {
+	background-position: -196px -44px;
+	height: 24px;
+	width: 20px;
+}
+.flair-hong-kong-attitude, .md a[href*="hong-kong-attitude"] {
+	background-position: -191px -71px;
+	height: 21px;
+	width: 24px;
+}
+.flair-machi-esports, .md a[href*="machi-esports"] {
+	background-position: -181px -94px;
+	height: 22px;
```

Comment:
Would you mind removing any occurrences of `height: 22px` or `width: 22px` from the new flairs? `.md a[href="#team"]` on L2261 sets that height and width as default anyway